CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+## 4.9.0 / TBA
+
+### ✨ New Features
+
+- New Logbook content: A logbook is a type of content that features logbook entries. It displays these entries chronologically as a timeline, with an emphasis on the time and the order.
```

Comment:
- New content: Logbook. A logbook is a type of content that features logbook entries. These entries are displayed as a timeline, with an emphasis on the time and the order, most recent at the top.